pywbem_mock/_namespaceprovider.py

```diff
@@ -85,16 +87,14 @@ def __init__(self, cimrepository):
         """
         super(CIMNamespaceProvider, self).__init__(cimrepository)
 
-        # NOTE: conn is only required for the following operation and
-        # is not kept in the constructed object.
-
         if not self.find_interop_namespace():
             raise CIMError(
                 CIM_ERR_INVALID_PARAMETER,
-                _format("No Interop namespace found. "
-                        "Construction of CIM_Namespace provider aborted."
-                        "Namespaces found: ", ", ".
-                        join(self.interop_namespace_names)))
+                _format("Cannot create namespace provider (for class {0}): "
+                        "No Interop namespace found in the CIM repository. "
+                        "Interop namespaces searched: {1}",
```

Comment:
Maybe change searched to allowed.  I could go either way.

Reply:
NOISSUE: My logic was: If we find something, we have first searched it.